### DISCUSSION TITLE
Stack Exchange source

### DISCUSSION THREAD

**Discussion by JeremyJaydan** [dailydotdev/daily#396] [Feature Requests]
I could suggest Stack Exchange via the source suggestion form but I feel this might be a bit more complex than adding a single RSS feed.

Source(s) of the Stack Enchange that cover all of its sites one of which being stack overflow would be neat.

daily.dev could be a way to be more aware of questions to answer and a search engine to find answers.

What do you think?

**Comment by idoshamun** [comment]
Hi @JeremyJaydan,

This is an interesting suggestion. As you mentioned, simply adding the Q&A to our feed doesn't make much sense but it requires something more.
Currently, we focus on written articles, once we get this right, we might think about expanding.
Thanks a lot
